template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/appsettings.json

```diff
@@ -1,26 +1,56 @@
 ﻿{
-////#if (OrganizationalAuth || IndividualB2CAuth)
+////#if (OrganizationalAuth || IndividualAuth)
+//  "Kestrel": {
+//    "Endpoints": {
+//      "LocalhostHttps": {
+//        "Address": "127.0.0.1",
+//        "Port": "443",
+//        "Certificate": "HTTPS"
+//      }
+//    }
+//  },
 //  "Authentication": {
-//#if (OrganizationalAuth)
+//#if (IndividualLocalAuth)
+//    "Identity": {
+//      "Audience": "63F0DED9-6722-437C-B5EF-F0543F3CF7FE",
+//      "TokenRedirectUrn": "urn:self:aspnet:identity:integrated",
+//      "Issuer": "https:////localhost/09732173-2cef-46b7-83db-1334bcb079d3/v2.0/",
```

Comment:
The scheme's `//` became a double `////` here and elsewhere in this file and perhaps other files. Bad search & replace?